src/components/ProductImages/ProductImages.tsx

```diff
@@ -1,9 +1,38 @@
+import { useState } from 'react';
+import styles from './productImages.module.css';
+
 interface ProductImagesProps {
-  imageList: string[];
+  images: string[];
 }
 
-const ProductImages = (_: ProductImagesProps) => {
-  return <></>;
+const ProductImages = ({ images }: ProductImagesProps) => {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  const handleImageClick = (index: number) => {
+    setActiveIndex(index);
+  };
+
+  return (
+    <div>
+      <img
+        className={styles.detailImage}
+        src={images ? images[activeIndex] : ''}
+        alt={images[activeIndex]}
```

Comment:
images is an array so there is any cases for this go in else statement?

Reply:
Fixed